src/Settings/View/CarrierSettingsView.php

```diff
@@ -0,0 +1,228 @@
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $allowMondayDelivery
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $allowMorningDelivery
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $allowOnlyRecipient
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $allowPickupLocations
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $allowSameDayDelivery
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $allowSaturdayDelivery
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $allowSignature
+ * @property null                                                          $cutoffTime
+ * @property null                                                          $cutoffTimeSameDay
+ * @property \MyParcelNL\Pdk\Form\Model\Input\Select\DropOffDaySelectInput $dropOffDays
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $featureShowDeliveryDate
+ * @property \MyParcelNL\Pdk\Form\Model\Input\TextInput                    $priceEveningDelivery
+ * @property \MyParcelNL\Pdk\Form\Model\Input\TextInput                    $priceMorningDelivery
+ * @property \MyParcelNL\Pdk\Form\Model\Input\TextInput                    $priceOnlyRecipient
+ * @property \MyParcelNL\Pdk\Form\Model\Input\TextInput                    $pricePackageTypeDigitalStamp
+ * @property \MyParcelNL\Pdk\Form\Model\Input\TextInput                    $pricePackageTypeMailbox
+ * @property \MyParcelNL\Pdk\Form\Model\Input\TextInput                    $pricePickup
+ * @property \MyParcelNL\Pdk\Form\Model\Input\TextInput                    $priceSameDayDelivery
+ * @property \MyParcelNL\Pdk\Form\Model\Input\TextInput                    $priceSignature
+ * @property \MyParcelNL\Pdk\Form\Model\Input\TextInput                    $priceStandardDelivery
+ * @property \MyParcelNL\Pdk\Form\Model\Input\SelectInput                  $defaultPackageType
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $exportAgeCheck
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $exportInsured
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $exportInsuredAmount
+ * @property \MyParcelNL\Pdk\Form\Model\Input\SelectInput                  $exportInsuredAmountMax
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $exportInsuredForBe
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $exportExtraLargeFormat
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $exportOnlyRecipient
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $exportReturnShipments
+ * @property \MyParcelNL\Pdk\Form\Model\Input\ToggleInput                  $exportSignature
+ * @property \MyParcelNL\Pdk\Form\Model\Input\SelectInput $digitalStampDefaultWeight
```

Comment:
dit hoeft helemaal niet, dit is voor models. Bovendien klopt het niet want die view heeft helemaal nooit deze attributen